docs/Makefile

```diff
@@ -0,0 +1,177 @@
+# Makefile for Sphinx documentation
```

Comment:
No, I pulled this in from the template repo. It is needed in the `make docs` step of the Makefile. Whether it can be auto-generated or not I'm not sure but I pulled it from that repo. I think I'm just going to take the `make docs` out altogether here.